backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/PredicatesHandler.java

```diff
@@ -179,9 +259,13 @@ private String applyOperator(
       case OperatorEnum.Operator.NE_VALUE:
         return String.format("%s != %s", colName, valueBinding);
       case OperatorEnum.Operator.CONTAIN_VALUE:
-        return String.format("%s LIKE %s", colName, valueBinding);
+        return String.format(
+            "%s LIKE %s",
+            "lower(" + colName + ") ", Pattern.compile(valueBinding).toString().toLowerCase());
```

Comment:
Why do we need this?

Reply:
- let say if there is key like `KEY_1` and user will try to predicate with value in small case like `key` then it will not returning anything so for that we have to take that column in lower case.